CHANGES.rst

```diff
@@ -33,6 +33,10 @@ Security fixes
     rejects such requests, this cannot be exploited. Additionally, exploiting this problem requires
     user interaction: they would need to click on a password reset link which they never requested,
     and which points to a domain that does not match the one where Indico is running.
+Internationalization
+^^^^^^^^^^^^^^^^^^^^
+
+- New translation: Polish
```

Comment:
The placement of this got slightly screwed up when rebasing (move it in the 2.3.5 section)